screens/ClockIn.js

```diff
@@ -245,112 +244,112 @@ class ClockIn extends React.Component {
 
     return (
       <DismissKeyboard>
-        <View style={[styles.container_nocenterCnt]}>
+        <View style={[styles.container, { backgroundColor: theme.background }]}>
           <ScreenHeader backNavigation={true} title={t('timeCardTitle')} />
 
           <View style={{ flex: 3, justifyContent: 'center' }}>
             <View style={[styles.fieldContainer]}>
               <View style={{ flex: 2 }}>
-                <Text style={styles.fieldTitle}>{t('currentLocation')}</Text>
+                <Text style={[styles.fieldTitle, { color: theme.foreground }]}>{t('currentLocation')}</Text>
```

Comment:
so this approach seems like it needs to update a lot of component. I was wondering if it's possible to modify the style (e.g. fieldTitle) by adding the color and backgroundColor attributes in it and when theme toggle button is clicked, it will change the color and backgroundColor accordingly in styles.js.